### ISSUE TITLE
Connect new task & edit task to firestore.

### ISSUE THREAD

**Issue by espenotlo** [janitaroyseth/solveit#122]
**ConfigureTaskScreen**
new task:
call task service to store the new task to firebase, and the project service to update the project on firebase with the new task (NB: in firebase project.tasks are stored by foreign key (taskId).

edit task:
call task service to store the new task to firebase. No need to update the project in firestore here.

**Comment by espenotlo** [comment]
NB: Dependent on #125.
